### ISSUE TITLE
[Idea] Filters - show selected values

### ISSUE THREAD

**Issue by mklahorst** [Laravel-Backpack/ideas#177]

**Comment by tabacitu** [comment]
Update: Just added this to the roadmap 🎉 We expect it'll be ready in Sep-Oct 2022!

**Comment by Mte90** [comment]
I see that the issue is still there after all those years.

**Comment by pxpm** [comment]
> I see that the issue is still there after all those years.

Hey @Mte90 it's not an issue, it's a feature request 👍 

It had been postponed because filters need a "major" rework, and when that happen I am pretty sure this is one of the scenarios that will be unblocked without having to write a bunch of "javascript patches" to workaround the current limitations. 

Since you replied to the thread you are now probably subscribed, so you will be getting updated when we work on this 👍 

Cheers
